ksql-engine/src/test/java/io/confluent/ksql/testutils/ExpressionParseTestUtil.java

```diff
@@ -0,0 +1,25 @@
+package io.confluent.ksql.testutils;
+
+import io.confluent.ksql.metastore.MetaStore;
+import io.confluent.ksql.parser.DefaultKsqlParser;
+import io.confluent.ksql.parser.KsqlParser;
+import io.confluent.ksql.parser.KsqlParser.ParsedStatement;
+import io.confluent.ksql.parser.KsqlParser.PreparedStatement;
+import io.confluent.ksql.parser.tree.Expression;
+import io.confluent.ksql.parser.tree.Query;
+import io.confluent.ksql.parser.tree.SingleColumn;
+
+public final class ExpressionParseTestUtil {
```

Comment:
Why is this different from `KsqlParserTestUtil`? I think I haven't wrapped my head around the split you've been working on.

Reply:
I didn't know about `KsqlParserTestUtil`. It looks like that parses a full statement, where what I want here is to parse just an expression. Let me look into combining the two.